pg_spot_operator/cmdb.py

```diff
@@ -528,7 +508,7 @@ def finalize_instance_setup(m: InstanceManifest):
 
     m.decrypt_secrets_if_any()
 
-    connstr_private, connstr_public = update_instance_connect_info(m)
+    connstr_private, connstr_public = get_instance_connect_strings(m)
     if connstr_private:
         logger.info(
             "*** PRIVATE Postgres connect string *** - '%s'", connstr_private
```

Comment:
do we want to log the password here?

Reply:
True not a very logical place indeed, will look for a better place